subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -847,7 +847,7 @@
 
   /**
    * Creates a new task that will handle failure of this task.
-   * Early completion due to cancellation is not considered to be a failure.
+   * Early completion due to cancellation is not considered to be a failure so it will not be recovered.
```

Comment:
and the recovery function is not called. (to keep it consistent with the next statement)?

Reply:
Will change.